priv/script.sh

```diff
@@ -2,441 +2,490 @@
 
 set -eu
 
+elixir_version=1.17.3-otp-27
```

Comment:
I moved these to the top of the script because they are the things most likely to be updated frequently. I have also updated them to the latest versions for Elixir and OTP.